data/xml/2021.jeptalnrecital.xml

```diff
@@ -0,0 +1,626 @@
+      <language>fra</language>
+    </paper>
+    <paper id="4">
+      <title>Caractérisation des relations sémantiques entre termes multi-mots fondée sur l’analogie (Semantic relations recognition between multi-word terms by means of analogy )</title>
+      <author><first>Yizhe</first><last>Wang</last></author>
+      <author><first>Béatrice</first><last>Daille</last></author>
+      <author><first>Nabil</first><last>Hathout</last></author>
+      <pages>34–43</pages>
+      <abstract>La terminologie d’un domaine rend compte de la structure du domaine grâce aux relations entre ses termes. Dans cet article, nous nous intéressons à la caractérisation des relations terminologiques qui existent entre termes multi-mots (MWT) dans les espaces vectoriels distributionnels. Nous avons constitué un jeu de données composé de MWT en français du domaine de l’environnement, reliés par des relations sémantiques lexicales. Nous présentons une expérience dans laquelle ces relations sémantiques entre MWT sont caractérisées au moyen de l’analogie. Les résultats obtenus permettent d’envisager un processus automatique pour aider à la structuration des terminologies.</abstract>
+      <url hash="06b23729">2021.jeptalnrecital-taln.4</url>
+      <language>fra</language>
+    </paper>
+    <paper id="5">
+      <title>Construire des ressources collaboratives pour les langues peu dotées: une modélisation orientée communauté (Building collaborative resources for poorly endowed languages : community-oriented modeling )</title>
+      <author><first>Elvis</first><last>Mboning</last></author>
+      <author><first>Ornella</first><last>Wandji</last></author>
+      <pages>44–52</pages>
+      <abstract>Les applications du traitement automatique des langues (TAL) nourrissent aujourd’hui une bonne partie des langues indo-européennes en raison des corpus linguistiques de qualité disponibles en grande quantité et variété. Les corpus de données open sources en langues africaines étant quasi inexistants, comment arrimer les avancées du TAL à ces langues peu dotées ? Dans cet article, nous examinons le problème de construction des ressources lexicographiques pour les langues peu dotées. Nous souhaitons introduire un modèle de construction des ressources lexicographiques en exploitant les compétences socio-linguistiques des communautés linguistiques locales. Au fil des sections, nous présenterons le nouveau modèle de codification des dictionnaires issue de cette modélisation orientée communauté.</abstract>
+      <url hash="1518edbc">2021.jeptalnrecital-taln.5</url>
+      <language>fra</language>
+    </paper>
+    <paper id="6">
+      <title>Contribution d’informations syntaxiques aux capacités de généralisation compositionelle des modèles seq2seq convolutifs (Assessing the Contribution of Syntactic Information for Compositional Generalization of seq2seq Convolutional Networks)</title>
+      <author><first>Diana</first><last>Nicoleta Popa</last></author>
```

Comment:
last name Popa

Reply:
@danielgildea Thanks!
Should I keep `Nicoleta` in the last name for first name?

Reply:
I think the first name is "Diana Nicoleta"

Reply:
NVM I figured it out